src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -139,13 +142,23 @@ export default class CodeSnippet extends Component {
         <span className={`${prefix}--snippet-btn--text`}>
           {expandCodeBtnText}
         </span>
-        <Icon
-          aria-hidden="true"
-          alt={expandCodeBtnText}
-          name="chevron--down"
-          description={expandCodeBtnText}
-          className={`${prefix}--icon-chevron--down`}
-        />
+        {componentsX ? (
+          <ChevronDown16
+            aria-hidden={true}
+            aria-label={expandCodeBtnText}
```

Comment:
Do we need the `aria-label` if `aria-hidden` is set to `true`? I bet we could use the content from the previous span for the `button` instead of using the icon.